pulsar-client-cpp/include/pulsar/MessageBuilder.h

```diff
@@ -94,8 +95,10 @@ class MessageBuilder {
  private:
     MessageBuilder(const MessageBuilder&);
     void checkMetadata();
-
+    static boost::shared_ptr<MessageImpl> getMessageImpl();
```

Comment:
@merlimat - static because the method gets data from objectpool which is a global variable, I guess the name is confusing - will change it to createMessageImpl